server/public/model/metrics.go

```diff
@@ -70,7 +70,7 @@ func (r *PerformanceReport) IsValid() error {
 	}
 
 	now := time.Now().UnixMilli()
-	if r.End < now-performanceReportTTLMilliseconds {
+	if r.End < float64(now-performanceReportTTLMilliseconds) {
```

Comment:
Out of the scope of the PR, but the thought came when working with this:

What if the client has a skew in time with the server? Wouldn't this fail?

Reply:
Yeah, if the skew is greater than 5 minutes, it would. I think that's not very likely to occur though, and if it was to happen, you'd see this error in the logs